travis/docker-compose.travis.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.7"
+
+services:
+
+  # A postgres 9.6 server for all the test data
+  postgres:
+    image: postgres:9.6
+    ports:
+      - "5433:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+
+  # This container is used to run the python test suite against the wagtail site
+  wagtail:
+    image: $DOCKER_REPO:$PYTHON_TAG
+    ports:
+      - "8000:8000"
+    env_file:
+      - ./travis.env
+    depends_on:
+      - postgres
+
+  # This container runs a version of the wagtail app that has mounted volumes we need to enable testing with cypress
+  wagtail_cypress:
+    image: $DOCKER_REPO:$PYTHON_TAG
+    ports:
+      - "8000:8000"
+    command: python network-api/manage.py runserver 0.0.0.0:8000
+    env_file:
+      - ./travis.env
+    volumes:
+      - static_files:/app/network-api/staticfiles/
+      - media_files:/app/network-api/media/original_images/
+    depends_on:
+      - postgres
+
+  # This container runs the visual regression testing using percy and cypress
+  cypress:
+    build:
+      context: ../
+      cache_from:
+        - $DOCKER_REPO:$CYPRESS_TAG
+      dockerfile: "./travis/Dockerfile.cypress"
+    image: $DOCKER_REPO:$CYPRESS_TAG
+    environment:
+      - CYPRESS_baseUrl=http://0.0.0.0:8000
+      - PERCY_TOKEN=$PERCY_TOKEN
+      - PERCY_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+      - PERCY_COMMIT=$TRAVIS_PULL_REQUEST_SHA
+      - PERCY_PULL_REQUEST=$TRAVIS_PULL_REQUEST
+    command: ./wait-for-it.sh -t 30 0.0.0.0:8000 -- /node_modules/.bin/percy exec -t 200 -- /node_modules/.bin/cypress run
```

Comment:
we just bumped this up from 200 to 800 because (surprise!) the wait wasn't long enough